scripts/convert_parakeet_gguf.py

```
@@ -0,0 +1,413 @@
+        return "joint.pred." + s[len("joint.decoder_joint.0."):]
+    if s.startswith("joint.pred_to_joint_network."):
+        return "joint.pred." + s[len("joint.pred_to_joint_network."):]
+
+    # ---- encoder pooler / projector (the enc_out projection) ----
+    # transformers: encoder.projector.weight / encoder.proj.weight
+    # This maps to joint.enc.weight in the C++ engine (the joint does the
+    # encoder projection). Actually, looking at the C++ code, the pooler is
+    # handled by joint.enc.weight — so this may already be covered above.
+    # If there's a separate pooler, it might be an extra tensor we don't need.
+    # Skip it for now.
+    if s.startswith("encoder.projector.") or s.startswith("encoder.proj."):
```

Ruff (0.16.1)

[warning] Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)